src/sentry/conf/server.py

```diff
@@ -77,8 +77,28 @@ def env(key, default='', type=None):
 # This gives access to functionality like the debug toolbar.
 INTERNAL_IPS = ()
 
+# List of IP subnets which should not be accessible
+# http://en.wikipedia.org/wiki/Reserved_IP_addresses
+SENTRY_DISALLOWED_IPS = (
```

Comment:
I am not sure this is such a good idea now as this will block all requests to GitHub Enterprise, GitLab On-premise etc. for Sentry On-Premise installs during integrations. I think this almost exclusively applies to getsentry so I'll remove this part.

Will still fix some Snuba/system `safe_urlopen` calls tho in a follow up.